authorization.jl

```diff
@@ -37,16 +37,16 @@ end
 
 function AuthorizeSpotify(client_id, client_secret)
     sc = SpotifyCredentials(client_id = client_id, client_secret = client_secret)
+    sc = base64encode(sc)
+    sc = GetAuthorizationToken(sc)
     return sc
 end
 
+#= 
```

Comment:
What is #=

Reply:
not sure, decent code but using is typed out in a not so nice to look at way